releasenotes/notes/trotter-qrte-primitives-8b3e495738b57fc3.yaml

```
@@ -0,0 +1,12 @@
+---
+features:
+  - |
+    Added :class:`qiskit.algorithms.time_evolvers.trotterization.TrotterQRTE` with
+    :class:`qiskit.primitives.BaseEstimator` as ``init`` parameter. It replaces
+    :class:`qiskit.algorithms.evolvers.trotterization.TrotterQRTE`.
```

```suggestion
    :class:`qiskit.algorithms.TrotterQRTE`.
```
The full name does not resolve to a link in the html - we document it at the level of algorithms so this should